src/config.hpp

```diff
@@ -285,6 +285,15 @@ class Config {
         return result->operator[]("metrics_enabled").as<bool>();
     }
 
+    /**
+     * @brief Check if result was parsed
+     *
+     * @return bool
+     */
+    bool isResultParsed() {
+        return result.get() == nullptr;
```

Comment:
```suggestion
        return result;
```
Should be sufficient

Reply:
Done.